.buildkite/fpm-pipeline.yml

```diff
@@ -1,3 +1,29 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/buildkite/pipeline-schema/main/schema.json
+
+env:
+  SETUP_GVM_VERSION: "v0.5.1"
+  GOLANG_VERSION: "1.21.3"
```

Comment:
If possible, I'd say to use the `.go-version` approach so this env variable can be loaded dynamically with the `hooks` and the [golang autobump](https://github.com/elastic/golang-crossbuild/blob/main/.ci/bump-golang.yml) can be updated and simplify since it uses the GO_VERSION in the [Jenkinsfile](https://github.com/elastic/golang-crossbuild/blob/bad301b60614ba06e7ea5333357038e9ea35d396/.ci/bump-go-release-version.sh#L28).

Reply:
Sure, thanks for the tip! Will add it.

Reply:
```suggestion
```